docs/source/admin/traffic_ops_using.rst

```diff
@@ -591,6 +592,20 @@ Parameters in the Mid (parent) profile that influence this feature:
 | use_ip_address | parent.config | 0       | 1 means use IP(v4) address of this parent in the parent.config, 0 means use the host_name.domain_name concatenation.                                                              |
 +----------------+---------------+---------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+.. _rl-qstring-handling:
+
+Qstring Handling
+++++++++++++++++
+
+Delivery services have a Query String Handling option that, when set to ignore, will automatically add a regex remap to that delivery service's config.  There may be times this is not preferred, or there may be requirements for one delivery service or server(s) to behave differently.  When this is required, the psel.qstring_handling parameter can be set in either the delivery service profile or the server profile, but it is important to note that the server profile will override ALL delivery services assigned to servers with this profile parameter.  If the parameter is not set for the server profile but is present for the Delivery Service profile, this will override the setting in the delivery service.  A value of "ignore" will not result in the addition of regex remap configuration.
+
```

Comment:
can you wrap this at a reasonable width?  A single space change would show the whole line as different..